spec/lib/miq_expression_spec.rb

```diff
@@ -1701,58 +1701,6 @@
       expect(filter.to_ruby).to eq('<value ref=vm, type=integer>/virtual/used_disk_storage</value> >= 1048576000')
     end
 
-    context "integration" do
-      it "should escape strings" do
-        filter = YAML.load '--- !ruby/object:MiqExpression
-        exp:
-          INCLUDES:
-            field: Vm.registry_items-data
-            value: $foo
-        '
-        expect(filter.to_ruby).to eq("<value ref=vm, type=text>/virtual/registry_items/data</value> =~ /\\$foo/")
-
-        data = {"registry_items.data" => "C:\\Documents and Users\\O'Neill, April\\", "/virtual/registry_items/data" => "C:\\Documents and Users\\O'Neill, April\\"}
```

Comment:
DO we have tests for this elsewhere?  Showing the escaping still happens seems like a valid test.

Reply:
nvm - just needed to scroll further :D

Reply:
all 3 of these tests were moved over to Condition. though I guess they changed a little as I removed the yaml and instead just used straight up `MiqExpression` syntax